samples/SDP/offers/1v1a1d-chrome-linux.txt

```diff
@@ -0,0 +1,144 @@
+v=0
```

Comment:
Yeah, I'll move them into a new test folder. Was a bit timid about adding a folder for test artifacts.

Reply:
FYI, I continued an assumption that the build dir is one directory under the project root, and that tests are run from that directory. That may be a bad assumption, but I noticed that RtpFunctionalityTest.cpp is already assuming this.

We could change this at some point (at least the build directory location assumption) by adding a cmake target to copy over the files into the build directory, so it could at least be agnostic to where the build dir lives (e.g. some editors put their build dirs at the same level as the project directory, instead of under it...)

Reply:
The other alternative is to simply define these as static const strings in test cpp